libs/entitlement/src/main/java/org/elasticsearch/entitlement/bootstrap/EntitlementBootstrap.java

```diff
@@ -22,15 +22,29 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
 
 public class EntitlementBootstrap {
 
+    public static Map<Path, Boolean> pluginData;
+    public static Function<Class<?>, String> pluginResolver;
+
     /**
      * Activates entitlement checking. Once this method returns, calls to forbidden methods
```

Comment:
Nit: While not modified as part of this PR, I wonder if forbidden should be changed to some other synonym because of the overload with forbidden method checking as part of style.